scripts/generate-app-config/generateEvidenceConfig.js

```diff
@@ -56,6 +56,7 @@ const createEvidencePage = (packageNamespace, locale, name) => (
 const generateEvidenceConfig = (generatePagesOptions, namespace) => {
   const patterns = getScreenshotPatterns(generatePagesOptions);
 
+  console.log(patterns);
```

Comment:
whoops! Remove the console log. :)

Reply:
yep this wasn't supposed to be left behind.